### PR TITLE
Update assembler of SVM and KMeans to use DXO

### PR DESCRIPTION
**Pull request by ZiyueXu77**
Fixes # .

### Description

Change the behavior to using DXO directly following the update from FedCE to the Assembler class

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

**Review by holgerroth — DISMISSED** [review]
LGTM

File: examples/advanced/sklearn-kmeans/jobs/sklearn_kmeans_base/app/custom/kmeans_assembler.py [hunk 1]
```diff
@@ -33,7 +32,8 @@ def __init__(self):
         self.count = None
         self.n_cluster = 0
 
-    def get_model_params(self, data: dict):
+    def get_model_params(self, dxo: DXO):
+        data = dxo.data
         return {"center": data["center"], "count": data["count"]}
 
     def assemble(self, data: Dict[str, dict], fl_ctx: FLContext) -> dict:
```
Comment (YuanTingHsieh) [inline examples/advanced/sklearn-kmeans/jobs/sklearn_kmeans_base/app/custom/kmeans_assembler.py:39]: please update this method signature to

`def assemble(self, data: Dict[str, dict], fl_ctx: FLContext) -> DXO:`

for all child class and parent class in nvflare/app_common/aggregators/assembler.py

Thanks

File: examples/advanced/sklearn-svm/jobs/sklearn_svm_base/app/custom/svm_assembler.py [hunk 2]
```diff
@@ -33,7 +32,8 @@ def __init__(self, kernel):
         self.support_y = None
         self.kernel = kernel
 
-    def get_model_params(self, data: dict):
+    def get_model_params(self, dxo: DXO):
+        data = dxo.data
         return {"support_x": data["support_x"], "support_y": data["support_y"]}
 
     def assemble(self, data: Dict[str, dict], fl_ctx: FLContext) -> dict:
```
Comment (YuanTingHsieh) [inline examples/advanced/sklearn-svm/jobs/sklearn_svm_base/app/custom/svm_assembler.py:39]: please update this method signature to

`def assemble(self, data: Dict[str, dict], fl_ctx: FLContext) -> DXO:`

for all child class and parent class in nvflare/app_common/aggregators/assembler.py

Thanks

Comment (ZiyueXu77) [reply examples/advanced/sklearn-svm/jobs/sklearn_svm_base/app/custom/svm_assembler.py:39]: ah yes, thanks!

File: examples/advanced/sklearn-kmeans/jobs/sklearn_kmeans_base/app/custom/kmeans_assembler.py (hunk 1 above)
Comment (ZiyueXu77) [reply examples/advanced/sklearn-kmeans/jobs/sklearn_kmeans_base/app/custom/kmeans_assembler.py:39]: updated

[the remark this answers is not in the document]
Comment (ZiyueXu77) [reply examples/advanced/sklearn-kmeans/jobs/sklearn_kmeans_base/app/custom/kmeans_assembler.py]: thanks! corrected both

**Review by YuanTingHsieh — APPROVED** [review]
thanks. LGTM

**Pull request merged**